peps/pep-0736.rst

```diff
@@ -0,0 +1,361 @@
+
+The syntax is ugly
+------------------
+
+This objection is by far the most common. On the contrary, we argue that:
+
+* This objection is is subjective and many community members disagree.
+* A nearly-identical syntax is already established for f-strings.
+* Programmers will, as ever, adjust over time.
+
+The feature is confusing
+------------------------
+
+We argue that:
+
+* Introducing new features typically has this impact temporarily.
+* The syntax is very similar to the established ``f'{x=}'`` syntax.
+* The feature and syntax are familiar from other popular modern languages.
+* The expansion of ``x=`` to ``x=x`` is in fact a trivial feature and inherently
+  significantly less complex than ``*arg`` and ``**kwarg`` expansion.
+* This particular syntactic form has been independently proposed on numerous
+  occasions, indicating that it is the most obvious [8]_ [9]_ [13]_.
+
+The feature is not explicit
+---------------------------
+
+This is based on a misunderstanding of the Zen of Python. Keyword arguments are
+fundamentally more explicit than positional ones where argument assignment is
+only visible at the function definition. On the contrary, the proposed syntactic
+sugar contains all the information as is conveyed by the established keyword
+argument syntax but without the redundancy. Moreover, the introduction of this
+syntactic sugar incentivises use of keyword arguments, making typical Python
+codebases more explicit.
+
+The feature adds another way of doing things
+--------------------------------------------
+
+The same argument can be made against all syntax changes. This is a simple
+syntactic sugar, much as ``x += 1`` is sugar for ``x = x + 1`` when ``x`` is an
+integer. This isn't tantamount to a 'new way' of passing arguments but a more
+readable notation for the same way.
+
+Renaming the variable in the calling context will break the code
+----------------------------------------------------------------
+
+A ``NameError`` would make the mistake abundantly clear. Moreover, text editors
+could highlight this based on static analysis ‒ ``f(x=)`` is exactly equivalent
+to writing ``f(x=x)``. If ``x`` does not exist, modern editors have no problem
+highlighting the issue.
+
+Recommendations
+===============
+
+As with any other language feature, the programmer should exercise their own
+judgement about whether to use it in any given context. We do not recommend
+enforcing a rule to use the feature in all cases where it may be applicable.
+
+Reference Implementation
+========================
+
+`A proposed implementation <https://github.com/Hels15/cpython/tree/last-build>`_
+for cpython has been provided by @Hels15.
+
+References
+==========
+
+.. [1] Issue 36817: Add = to f-strings for easier debugging. - Python tracker
+   https://bugs.python.org/issue36817
+.. [2] Ruby keyword argument syntactic sugar
+   https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/#:~:text=Other%20Notable%20New%20Features
```

Comment:
Are these highlights intentional? If not, let's delete. (I'm not sure if they're Chrome only?) 
```suggestion
   https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/
```

Reply:
It is intentional. It looks to me like it has broad browser support as well https://caniuse.com/url-scroll-to-text-fragment

Reply:
I'm using Firefox, and the highlight didn't do anything for me. It might be better to deep link to the nearest linkable section heading?

Reply:
Yes, caniuse.com indicates that unfortunately Firefox is the only major browser lacking support yet. The ruby-lang.org doesn't  have any IDs I can anchor to so my sense is to leave this one is as it's helpful where it does work? For reasonml.github.io, I'll switch to #function-application

Reply:
For pages without linkable anchors, I tend to add a hint in the reference on what word or words to search for in the page, like this: [Ruby 3.1.0 release notes](https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/#:~:text=Other%20Notable%20New%20Features) (search for "keyword arguments").

Reply:
Sounds like a good idea, will do.